spec/models/journal_entry_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+RSpec.describe JournalEntry, type: :model do
+
+  journal_test = Journal.new
```

Comment:
Can you try using FactoryBot.build ?

Reply:
Is it necessary to user .build?

I solved it with FactoryBot.create because it is done like that in our sample project aswell